Removes parenthesis around replaced params

frosso opened the pull request:
Fixes https://github.com/kuflash/react-router-sitemap/issues/43
Old:
```
/route(/:param) {param: 'value'} => /route(/value)
/route(/:param) {} => /route(/:param)
```

With fix:
```
/route(/:param) {param: 'value'} => /route/value
/route(/param) {} => /route
```

LMK if you'd like me to add tests or stuff :)

kuflash commented:
@frosso cool! Yes, please add tests for this is cases. And can you create PR to development branch?

frosso commented:
Sure can!

frosso commented:
I added a couple of tests.
Usually I would add the tests before the change, but 🤷‍♂️  :D

Anyway, to make sure my tests were accurate, I removed my changes, wrote the tests, ran the tests and saw them fail. 
Then I re-added my changes and the tests were passing.

Let me know if I should add anything else :)

kuflash commented:
@frosso thank you, very much!

frosso commented:
@kuflash not a problem, thank *you*!
